doc/development.md

```diff
@@ -26,14 +32,8 @@ Create virtual environment for vdsm:
     pip install -r docker/requirements.txt
     deactivate
 
-Before running vdsm tests, activate the environment:
-
-    source ~/.venv/vdsm/bin/activate
-
-When done, you can deactivate the environment:
-
-    deactivate
-
+Read [Vdsm tests](../tests/README.md) for more information on
+how to run tests.
```

Comment:
I don't think we need this sentence, the next topic is running the tests, and this is a pretty small document.